index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <meta property="og:title" content="Linkbrary">
+  <meta property="og:url" content="WITHNJ.NETLIFY.APP">
```

Comment:
프로토콜을 포함한 링크를 넣어주어야 합니다! 그리고 특별한 이유가 없다면 대문자보단 소문자로 하는것이 좋을것 같아요 👍 
```suggestion
  <meta property="og:url" content="https://withnj.netlify.app">
```